pkg/controller/gslb/gslb_controller_test.go

```diff
@@ -525,6 +627,126 @@ func TestGslbController(t *testing.T) {
 	})
 }
 
+func createHealthyService(t *testing.T, serviceName string, cl client.Client, gslb *ohmyglbv1beta1.Gslb) {
+
```

Comment:
Please add `t.Helper()` so the stacktrace will report specific failing test context, not just function limited scope.

See https://golang.org/pkg/testing/#T.Helper